ui/components/ui/logo/logo-coinbasepay.scss

```diff
@@ -0,0 +1,3 @@
+[data-theme=dark] .logo-coinbasepay path {
+    fill: #fff;
```

Comment:
@georgewrmarshall What var should we use here?

Reply:
Updated!